net/third_party/quiche/src/quiche/common/platform/api/quiche_file_utils.cc

```diff
@@ -41,11 +42,16 @@ bool EnumerateDirectoryRecursivelyInner(absl::string_view path,
   }
   return true;
 }
+#endif  // !defined(STARBOARD)
 
 bool EnumerateDirectoryRecursively(absl::string_view path,
                                    std::vector<std::string>& files) {
+#if defined(STARBOARD)
+  return EnumerateDirectoryRecursivelyImpl(path, files);
+#else
   constexpr int kRecursionLimit = 20;
   return EnumerateDirectoryRecursivelyInner(path, kRecursionLimit, files);
```

Comment:
Would you create a backlog bug and add it in a TODO here to try to share more functionality with upstream? Definitely not that important but would be nice